docs/section22/Rule22-15.md

```diff
@@ -0,0 +1,52 @@
+
+# CHW&CW - Rule 22-15  
+
+**Schema Version:** 0.0.25    
+**Mandatory Rule:** True  
+**Rule ID:** 22-15  
+**Rule Description:** Heat Rejection Device Approach calaculated correctly (T/F), Approach = 25.72-(0.24*WB)
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** G3.1.3.11
+**Data Lookup:** None  
+**Evaluation Context:** Each HeatRejection 
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, 13, 7b, 8b, 12b.
+2. B-RMR is modeled with heat rejection loop
+
+**Function Call:**  
+
+1. get_heat_rejection_loops_connected_to_baseline_systems()
+
+**Applicability Checks:**  
+
+- The function get_heat_rejection_loops_connected_to_baseline_systems() returns a list of loops that are connected to Type 7,8,12,13,7b,8b,12b.  Get the list of applicable heat rejection loops: `heat_rejection_loop_ids = get_heat_rejection_loops_connected_to_baseline_systems(B_RMI)`
+
+- check applicability for each HeatRejection in the model: `for heat_rejection in B_RMI.heat_rejections:`
+
+  - check if heat_rejection fluid_loop is one of the applicable loops: `if heat_rejection.loop.id in heat_rejection_loop_ids:`
+  
+    - check if the design wetbulb temperature is between 55 and 90F (inclusive): `if ( heat_rejection.design_wetbulb_temperature >= 55 ) AND ( heat_rejection.design_wetbulb_temperature <= 90 ): CHECK_RULE_LOGIC`
+
+    - Else, rule is not applicable to the heat rejection: `else: RULE_NOT_APPLICABLE`
+
+  - Else, rule is not applicable to the heat rejection: `else: RULE_NOT_APPLICABLE`
```

Comment:
Should we account for multiple heat rejection loops? Otherwise looks good.

Reply:
line 26 looks at each heat rejection loop.  You'll notice that the rule logic is indented so that it is within the loop created on line 26, so we are looking at every heat rejection in the RMI